src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/config/HmppsTemplateKotlinExceptionHandler.kt

```diff
@@ -49,17 +48,6 @@ class HmppsTemplateKotlinExceptionHandler {
       ),
     ).also { log.debug("Forbidden (403) returned: {}", e.message) }
 
-  @ExceptionHandler(ExampleErrorHandlingNotFoundException::class)
-  fun handleException(e: ExampleErrorHandlingNotFoundException): ResponseEntity<ErrorResponse> = ResponseEntity
```

Comment:
decided to simplify template by removing extra endpoint that was only called for demo purposes and was wiremocked out anyway.  Therefore didn't need this exception handling